Description: Fix using *FLAGS in the Makefile, add gtk+-2.0 libs to LIBS
Author: Andrey Rahmatullin <wrar@wrar.name>
Last-Update: 2013-06-25

--- a/Makefile
+++ b/Makefile
@@ -11,20 +11,20 @@ CFLAGS += -Wall -fPIC `pkg-config gtk+-2.0 --cflags` -DPACKAGE="\"gkrellmpc\""
 ifeq ($(enable_nls),1)
 	CFLAGS += -DENABLE_NLS
 endif
-LDFLAGS += `curl-config --libs`
+LIBS += `pkg-config gtk+-2.0 --libs` `curl-config --libs`
 
 .PHONY: all clean dist install install_lib install_local_lib install_home install_instructions deinstall uninstall test strip
 
 all: gkrellmpc.so
 
 gkrellmpc.so: $(OBJECTS)
-	$(CC) $(CFLAGS) $(LDFLAGS) -shared -o gkrellmpc.so $(OBJECTS)
+	$(CC) $(LDFLAGS) -shared -o gkrellmpc.so $(OBJECTS) $(LIBS)
 ifeq ($(enable_nls),1)
 	(cd po && ${MAKE} all)
 endif
 
 %.o: %.c %.h globals.h
-	$(CC) $(CFLAGS) -c -o $@ $<
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $<
 
 strip: gkrellmpc.so
 	strip gkrellmpc.so
